src/vcConvert.cpp

```diff
@@ -398,7 +398,7 @@ void vcConvert_ShowUI(vcState *pProgramState)
   }
   else
   {
-    if ((pSelectedJob->status == vcCQS_Preparing || pSelectedJob->status == vcCQS_Cancelled))
+    if (pSelectedJob->status == vcCQS_Preparing || (pSelectedJob->status >= vcCQS_Cancelled && pSelectedJob->status <= vcCQS_Failed))
```

Comment:
I'm not a huge fan of this range check, it goes from a non-error condition to an error condition. I can only see problems arising from this kind of assumption.

Reply:
Also can cause non-obvious errors if you modify the enum to add another value in that range